packages/pwa/app/components/_app/index.jsx

```diff
@@ -73,6 +78,7 @@ const App = (props) => {
     const history = useHistory()
     const location = useLocation()
     const authModal = useAuthModal()
+    const addToCartModal = useAddToCartModal()
```

Comment:
We can keep this logic away from the app component by creating a component. This also makes it consistent with the way we created `CategoriesProvider` and `CurrencyProvider` 
```
export const AddToCartModalProvider = ({children}) => {
    const addToCartModal = useAddToCartModal()
    return (
        <AddToCartModalContext.Provider value={addToCartModal}>
            <AddToCartModal />
            {children}
        </AddToCartModalContext.Provider>
    )
}
```